include/miopen/config.h.in

```diff
@@ -53,6 +53,7 @@
 #cmakedefine HIP_OC_COMPILER "@HIP_OC_COMPILER@"
 #cmakedefine MIOPEN_HIP_COMPILER "@MIOPEN_HIP_COMPILER@"
 #cmakedefine EXTRACTKERNEL_BIN "@EXTRACTKERNEL_BIN@"
+#cmakedefine OFFLOADBUNDLER_BIN "@OFFLOADBUNDLER_BIN@"
```

Comment:
This should probably be prefix with `MIOPEN_` to avoid conflicts with other libraries. I know we dont do that for `EXTRACTKERNEL_BIN`, but we probably should have.

Reply:
done